contrib/docker/testnet.sh

```diff
@@ -186,6 +197,12 @@ set_args()
     KOVRI_BIN_ARGS="--floodfill 1 --disable-su3-verification 1"
     read_input "Change kovri binary arguments? [KOVRI_BIN_ARGS=\"${KOVRI_BIN_ARGS}\"]" KOVRI_BIN_ARGS
   fi
+
+  # Set firewalled daemon binary arguments
+  if [[ ${KOVRI_NB_FW} -gt 0 && -z $KOVRI_FW_BIN_ARGS ]]; then
+    KOVRI_FW_BIN_ARGS="--floodfill 0 --disable-su3-verification 1"
```

Comment:
Should we enable i2pcontrol (as noted in entrypoint iptables)?

Reply:
It's not mandatory.
It's always open on iptables b/c it was not easily overwritable it in the entrypoint.
Perso, i enable by default i2pcontrol on all instances since if not used it doesn't consume ressources, and i'm currently setting up a monitoring system, so i hope to use it soon(tm)